.github/CHANGELOG.md

```diff
@@ -74,6 +74,8 @@
 1. [A32NX] Fixed appearance of FCU decals on MSFS2024 - @tracernz (Mike)
 1. [FMS] Fixed issue with airport loading timing out on MSFS2024 - @tracernz (Mike)
 1. [A32NX] Fixed APU fire detection - @tracernz (Mike)
+1. [A32NX/PFD] Syncronize flashing/pulsing components across PFD - @lukecologne (luke)
+1. [A32NX/PFD] Synchronize flashing/pulsing components across PFD - @lukecologne (luke)
```

Comment:
Duplicate change log lines now (first one with the typo).

Reply:
Fixed